Description: make sure symbols are exported from main binary to allow loading modules
Author: Tzafrir Cohen <tzafrir@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -403,6 +403,8 @@
 main_makefile=1
 include Makefile.defs
 
+$(MAIN_NAME): LDFLAGS += -Wl,--export-dynamic
+
 static_modules_path=$(addprefix modules/, $(static_modules))
 extra_sources=$(wildcard $(addsuffix /*.c, $(static_modules_path)))
 extra_objs=$(extra_sources:.c=.o)
